scripts/SANS/sans/gui_logic/presenter/run_tab_presenter.py

```diff
@@ -340,6 +340,10 @@ def set_view(self, view):
 
         self._view.setup_layout()
 
+        # provide an option for users to show or hide the plot-results checkbox from the properties file
+        # it also ensure the checkbox is hidden by default
```

Comment:
I think that this comment is covered by the documentation above the method and the method name, so I don't think it needs to be here.